src/Client/pages/WishList.fs

```diff
@@ -243,12 +243,27 @@ let newBookForm (model:Model) dispatch =
                         | None -> ()
                         | Some e -> yield p [ClassName "text-danger"][str e]
                     ]
-                ]                    
-            ]        
+                ]
+            ]
         ]
     ]
 
-let view (model:Model) (dispatch: Msg -> unit) = 
+type [<Pojo>] BookProps = { key: string; book: Book; removeBook: unit -> unit }
```

Comment:
@alfonsogarciacaro I wonder if we really need pojo everywhere?

Reply:
For Fable 1.x unfortunately yes. If you don't use it, Fable will create a JS class for the record and React will complain as it expects plain objects.

For Fable 2.0 it won't be needed as we're planning to compile records always as plain objects :)